kapitan/resources.py

```diff
@@ -231,7 +231,7 @@ def inventory(search_paths, target, inventory_path="inventory/"):
     return inventory_reclass(full_inv_path)["nodes"][target]
 
 
-def inventory_reclass(inventory_path):
+def inventory_reclass(inventory_path, targets_path=None):
     """
     Runs a reclass inventory in inventory_path
```

Comment:
Can you update this docstring as well to reflect the new behavior?

Reply:
alright!